webapp/channels/src/components/post_view/new_message_separator/new_message_separator.tsx

```diff
@@ -17,38 +17,45 @@ type Props = {
     threadId?: string;
 }
 
-export default class NewMessageSeparator extends React.PureComponent<Props> {
-    render(): JSX.Element {
-        const pluginItems = this.props.newMessagesSeparatorActions?.
-            map((item) => {
-                if (!item.component) {
-                    return null;
-                }
-
-                const Component = item.component as any;
-                return (
-                    <Component
-                        key={item.id}
-                        lastViewedAt={this.props.lastViewedAt}
-                        channelId={this.props.channelId}
-                        threadId={this.props.threadId}
-                    />
-                );
-            });
-
-        return (
-            <div
-                ref={this.props.wrapperRef}
-                className='new-separator'
-            >
-                <NotificationSeparator id={this.props.separatorId}>
-                    <FormattedMessage
-                        id='posts_view.newMsg'
-                        defaultMessage='New Messages'
-                    />
-                    {pluginItems}
-                </NotificationSeparator>
-            </div>
-        );
-    }
-}
+const NewMessageSeparator = ({
+    newMessagesSeparatorActions,
+    lastViewedAt,
+    channelId,
+    threadId,
+    wrapperRef,
+    separatorId,
+}: Props) => {
+    const pluginItems = newMessagesSeparatorActions?.
+        map((item) => {
+            if (!item.component) {
+                return null;
+            }
+
+            const Component = item.component as any;
+            return (
+                <Component
+                    key={item.id}
+                    lastViewedAt={lastViewedAt}
+                    channelId={channelId}
+                    threadId={threadId}
+                />
+            );
+        });
+
+    return (
+        <div
+            ref={wrapperRef}
```

Comment:
i dont think ref can be passed this way, dont we have to use forwardRef? cc @larkox